lib/rules/multiple-assertions-checker.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
```

Comment:
I've added support for duplicate error messages and also upgraded the code a bit so it's more readable.